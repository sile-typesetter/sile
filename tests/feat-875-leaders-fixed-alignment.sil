\begin[papersize=a5]{document}
\script[src=packages/leaders]
\nofolios
\neverindent
% Test case: \leaders[width=...] with a given fixed width.
\font[size=20pt]

% For reference, full-width "a" leaders
|\leaders{a}|

% Now our test for vertical alignment a fixed-width leaders
A \leaders[width=50%lw]{a} B

% For reference, full-width "/\" leaders
|\leaders{/\\}|

% Now our test for vertical alignment of several fixed-width leaders in the same line
A\leaders[width=50pt]{/\\}B\leaders[width=50pt]{/\\}C\leaders[width=50pt]{/\\}D\leaders[width=50pt]{/\\}E\leaders[width=50pt]{/\\}F.

\end{document}
