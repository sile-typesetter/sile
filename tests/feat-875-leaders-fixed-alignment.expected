Set paper size 	419.5275636	595.275597
Begin page
Mx 	20.9764
My 	46.0724
Set font 	Gentium Plus;20;400;;normal;;LTR
T	95 w=4.8828	(|)
Mx 	31.3637
T	68 w=9.1797	(a)
Mx 	40.5434
T	68 w=9.1797	(a)
Mx 	49.7231
T	68 w=9.1797	(a)
Mx 	58.9027
T	68 w=9.1797	(a)
Mx 	68.0824
T	68 w=9.1797	(a)
Mx 	77.2621
T	68 w=9.1797	(a)
Mx 	86.4418
T	68 w=9.1797	(a)
Mx 	95.6215
T	68 w=9.1797	(a)
Mx 	104.8012
T	68 w=9.1797	(a)
Mx 	113.9809
T	68 w=9.1797	(a)
Mx 	123.1606
T	68 w=9.1797	(a)
Mx 	132.3402
T	68 w=9.1797	(a)
Mx 	141.5199
T	68 w=9.1797	(a)
Mx 	150.6996
T	68 w=9.1797	(a)
Mx 	159.8793
T	68 w=9.1797	(a)
Mx 	169.0590
T	68 w=9.1797	(a)
Mx 	178.2387
T	68 w=9.1797	(a)
Mx 	187.4184
T	68 w=9.1797	(a)
Mx 	196.5981
T	68 w=9.1797	(a)
Mx 	205.7777
T	68 w=9.1797	(a)
Mx 	214.9574
T	68 w=9.1797	(a)
Mx 	224.1371
T	68 w=9.1797	(a)
Mx 	233.3168
T	68 w=9.1797	(a)
Mx 	242.4965
T	68 w=9.1797	(a)
Mx 	251.6762
T	68 w=9.1797	(a)
Mx 	260.8559
T	68 w=9.1797	(a)
Mx 	270.0356
T	68 w=9.1797	(a)
Mx 	279.2152
T	68 w=9.1797	(a)
Mx 	288.3949
T	68 w=9.1797	(a)
Mx 	297.5746
T	68 w=9.1797	(a)
Mx 	306.7543
T	68 w=9.1797	(a)
Mx 	315.9340
T	68 w=9.1797	(a)
Mx 	325.1137
T	68 w=9.1797	(a)
Mx 	334.2934
T	68 w=9.1797	(a)
Mx 	343.4731
T	68 w=9.1797	(a)
Mx 	352.6527
T	68 w=9.1797	(a)
Mx 	361.8324
T	68 w=9.1797	(a)
Mx 	371.0121
T	68 w=9.1797	(a)
Mx 	380.1918
T	68 w=9.1797	(a)
Mx 	393.6684
T	95 w=4.8828	(|)
Mx 	20.9764
My 	70.0724
T	36 w=11.9043	(A)
Mx 	40.5434
T	68 w=9.1797	(a)
Mx 	49.7231
T	68 w=9.1797	(a)
Mx 	58.9027
T	68 w=9.1797	(a)
Mx 	68.0824
T	68 w=9.1797	(a)
Mx 	77.2621
T	68 w=9.1797	(a)
Mx 	86.4418
T	68 w=9.1797	(a)
Mx 	95.6215
T	68 w=9.1797	(a)
Mx 	104.8012
T	68 w=9.1797	(a)
Mx 	113.9809
T	68 w=9.1797	(a)
Mx 	123.1606
T	68 w=9.1797	(a)
Mx 	132.3402
T	68 w=9.1797	(a)
Mx 	141.5199
T	68 w=9.1797	(a)
Mx 	150.6996
T	68 w=9.1797	(a)
Mx 	159.8793
T	68 w=9.1797	(a)
Mx 	169.0590
T	68 w=9.1797	(a)
Mx 	178.2387
T	68 w=9.1797	(a)
Mx 	187.4184
T	68 w=9.1797	(a)
Mx 	196.5981
T	68 w=9.1797	(a)
Mx 	205.7777
T	68 w=9.1797	(a)
Mx 	214.9574
T	68 w=9.1797	(a)
Mx 	232.3068
T	37 w=10.9961	(B)
Mx 	20.9764
My 	94.0724
T	95 w=4.8828	(|)
Mx 	41.9301
T	18 w=9.3848	(/)
Mx 	51.3149
T	63 w=9.3848	(\)
Mx 	60.6996
T	18 w=9.3848	(/)
Mx 	70.0844
T	63 w=9.3848	(\)
Mx 	79.4692
T	18 w=9.3848	(/)
Mx 	88.8539
T	63 w=9.3848	(\)
Mx 	98.2387
T	18 w=9.3848	(/)
Mx 	107.6235
T	63 w=9.3848	(\)
Mx 	117.0082
T	18 w=9.3848	(/)
Mx 	126.3930
T	63 w=9.3848	(\)
Mx 	135.7777
T	18 w=9.3848	(/)
Mx 	145.1625
T	63 w=9.3848	(\)
Mx 	154.5473
T	18 w=9.3848	(/)
Mx 	163.9320
T	63 w=9.3848	(\)
Mx 	173.3168
T	18 w=9.3848	(/)
Mx 	182.7016
T	63 w=9.3848	(\)
Mx 	192.0863
T	18 w=9.3848	(/)
Mx 	201.4711
T	63 w=9.3848	(\)
Mx 	210.8559
T	18 w=9.3848	(/)
Mx 	220.2406
T	63 w=9.3848	(\)
Mx 	229.6254
T	18 w=9.3848	(/)
Mx 	239.0102
T	63 w=9.3848	(\)
Mx 	248.3949
T	18 w=9.3848	(/)
Mx 	257.7797
T	63 w=9.3848	(\)
Mx 	267.1645
T	18 w=9.3848	(/)
Mx 	276.5492
T	63 w=9.3848	(\)
Mx 	285.9340
T	18 w=9.3848	(/)
Mx 	295.3188
T	63 w=9.3848	(\)
Mx 	304.7035
T	18 w=9.3848	(/)
Mx 	314.0883
T	63 w=9.3848	(\)
Mx 	323.4731
T	18 w=9.3848	(/)
Mx 	332.8578
T	63 w=9.3848	(\)
Mx 	342.2426
T	18 w=9.3848	(/)
Mx 	351.6274
T	63 w=9.3848	(\)
Mx 	361.0121
T	18 w=9.3848	(/)
Mx 	370.3969
T	63 w=9.3848	(\)
Mx 	393.6684
T	95 w=4.8828	(|)
Mx 	20.9764
My 	118.0724
T	36 w=11.9043	(A)
Mx 	41.9301
T	18 w=9.3848	(/)
Mx 	51.3149
T	63 w=9.3848	(\)
Mx 	60.6996
T	18 w=9.3848	(/)
Mx 	70.0844
T	63 w=9.3848	(\)
Mx 	82.8807
T	37 w=10.9961	(B)
Mx 	98.2387
T	18 w=9.3848	(/)
Mx 	107.6235
T	63 w=9.3848	(\)
Mx 	117.0082
T	18 w=9.3848	(/)
Mx 	126.3930
T	63 w=9.3848	(\)
Mx 	143.8768
T	38 w=10.7227	(C)
Mx 	173.3168
T	18 w=9.3848	(/)
Mx 	182.7016
T	63 w=9.3848	(\)
Mx 	204.5994
T	39 w=11.8555	(D)
Mx 	229.6254
T	18 w=9.3848	(/)
Mx 	239.0102
T	63 w=9.3848	(\)
Mx 	266.4549
T	40 w=9.9219	(E)
Mx 	285.9340
T	18 w=9.3848	(/)
Mx 	295.3188
T	63 w=9.3848	(\)
Mx 	304.7035
T	18 w=9.3848	(/)
Mx 	314.0883
T	63 w=9.3848	(\)
Mx 	326.3768
T	41 w=9.5605	(F)
Mx 	335.9373
T	17 w=4.5801	(.)
End page
Finish
